### PR TITLE
chore(datadog-agent): add pending-upstream-fix event for cve-2024-49750

### PR DESCRIPTION
**Pull request by maxgio92**
Add pending-upstream-fix event for CVE-2024-49750.

### CODE REVIEW

File: datadog-agent.advisories.yaml [hunk 1]
```diff
@@ -497,6 +497,10 @@ advisories:
             componentType: python
             componentLocation: /usr/share/datadog-agent/lib/python3.11/site-packages/snowflake_connector_python-3.12.1.dist-info/METADATA, /usr/share/datadog-agent/lib/python3.11/site-packages/snowflake_connector_python-3.12.1.dist-info/RECORD, /usr/share/datadog-agent/lib/python3.11/site-packages/snowflake_connector_python-3.12.1.dist-info/direct_url.json, /usr/share/datadog-agent/lib/python3.11/site-packages/snowflake_connector_python-3.12.1.dist-info/top_level.txt
             scanner: grype
+      - timestamp: 2024-11-06T11:09:36Z
+        type: pending-upstream-fix
+        data:
+          note: Upstream package repository doesn't contain yet snowflake_connector_python-3.12.3. See https://agent-int-packages.datadoghq.com/external/snowflake-connector-python/.
```
Comment (hectorj2f) [inline datadog-agent.advisories.yaml:503]: Do you mean it is not available as part of their hosted datadog dependencies ? If so, we might want to add that on the note.

Comment (maxgio92) [reply datadog-agent.advisories.yaml:503]: Yes exactly. Thanks, I'm going to make it more clear

Comment (maxgio92) [reply datadog-agent.advisories.yaml]: @hectorj2f PTAL. Is that more clear now? Thanks

**Review by hectorj2f — APPROVED** [review]

File: datadog-agent.advisories.yaml (hunk 1 above)
Comment (hectorj2f) [reply datadog-agent.advisories.yaml]: Thanks @maxgio92

**Review by philroche — APPROVED** [review]

**Pull request merged**